hi-ml-azure/src/health_azure/logging.py

```diff
@@ -45,6 +46,39 @@ def logging_to_stdout(log_level: Union[int, str] = logging.INFO) -> None:
     logger.setLevel(log_level)
 
 
+def logging_to_file(file_path: Path, log_level: Union[int, str] = logging.INFO) -> None:
+    """
+    Instructs the Python logging libraries to start writing logs to the given file.
+    Logging will use a timestamp as the prefix, using UTC. The logging level will be the same as defined for
```

Comment:
"The log level will be the same as defined for logging to stdout" -> unless overwritten here?